### PR TITLE
Added comment on README

### PR DESCRIPTION
**Pull request by morethanyell**
### Description of the changes:
-
-
-

### How changes were validated:

-
-
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,3 +62,9 @@ email to ensure we received your original message. Further information, includin
 Copyright (c) Microsoft Corporation. All rights reserved.
 
 Licensed under the [MIT License](./LICENSE).
+
+## Forker
+Daniel L. Astillero
+daniel.l.astillero@gmail.com
++63917-390-2559
```
Comment (johannesschobel) [inline README.md]: you sure you want to commit this to the offical repository from microsoft?

Comment (ohlookitsderpy) [reply README.md]: what even is this?

Comment (zack6849) [reply README.md]: a poor attempt at advertising himself

Comment (ohlookitsderpy) [reply README.md]: yikes

**Comment by ohlookitsderpy** [comment]
thanks matt

**Pull request closed without merging**